vnext/local-cli/runWindows/utils/build.js

```diff
@@ -63,16 +64,34 @@ async function nugetRestore(nugetPath, slnFile, verbose, msbuildVersion) {
 }
 
 async function restoreNuGetPackages(options, slnFile, verbose) {
-  const nugetPath =
+  let nugetPath =
     options.nugetPath || path.join(os.tmpdir(), 'nuget.4.9.2.exe');
 
-  const dlNugetText = 'Downloading NuGet Binary';
-  const ensureNugetSpinner = newSpinner(dlNugetText);
-  const exists = await existsAsync(nugetPath);
-  if (!exists) {
+  const ensureNugetSpinner = newSpinner('Locating NuGet executable');
+  if (!(await existsAsync(nugetPath))) {
+    try {
+      let localNuget = execSync('where nuget')
+        .toString()
+        .trim();
+      // Ensure the local nuget exe is at least version 4.6
```

Comment:
what is it about 4.6? maybe clarify in the comment?

Reply:
I was duplicating various things we've had with out CI loops requiring newer versions of nuget.  But looking through it, that might be specific to azure dev ops, and not apply to consumers.

There is also a requirement to have a new enough version to support variables during publish.  But again that might not be needed.  I'll remove the version check for now.